### ISSUE TITLE
Door close no longer working

### ISSUE THREAD

**Issue by brbeaird** [pfeffed/liftmaster_myq#1]
After the latest Chamberlain update, the door close command (desireddoorstate = 2) is no longer working. The API just throws a generic error. The open command (desireddoorstate = 1) still works fine. Any ideas on this?

Update: I figured it out I think. DesiredDoorState for close is achieved now by sending 0.

**Comment by pfeffed** [comment]
Yeah, they seem to have made a lot of changes to their APIs recently.  I'm
out of the country right now but will try to run some fresh packet captures
when I get home.  It probably be a couple of days before I can turn this
around.

David Maurice Pfeffer
652 Melville Avenue | Palo Alto, CA 94301 | USA
H: +1.650.331.0059 | M: +1.202.329.3557 | E: david@pfeffer.org

**Comment by jonedavis** [comment]
I've also experience some issues with setting the door state. My JSON looks like:

```
{
    "AttributeName":"desireddoorstate",
    "DeviceId":"xxxx",          
 "ApplicationId":"Vj8pQggXLhLy0WHahglCD4N1nAkkXQtGYpq2HrHD7H1nvmbT55KqtN6RSF4ILB%252fi",
    "AttributeValue":"0",
    "SecurityToken":"xxx"
}

```

And I always receive the follow error message:

```
{
    "ReturnCode": "217",
    "ErrorMessage": "Error processing request (217)",
    "CorrelationId": "cea60ab2-3336-42c9-bea5-9f8074f4e3ee"
}
```

Any insight to this would be greatly appreciated.
Thanks!
